patch/03.labels.patch

```diff
@@ -0,0 +1,270 @@
+diff --git a/templates/_helpers.tpl b/templates/_helpers.tpl
```

Comment:
Why is this needed?

Reply:
The original creator of this repository tried to sync up this chart with the upstream chart of gatekeeper. The script used for this is [here]https://github.com/giantswarm/gatekeeper-app/blob/master/script/sync_chart.sh). There are [other patches in the directory](https://github.com/giantswarm/gatekeeper-app/tree/master/patch)